manifests/flux-monitor.yaml

```diff
@@ -0,0 +1,51 @@
+apiVersion: monitoring.coreos.com/v1
+kind: ServiceMonitor
+metadata:
+  name: flux
+  namespace: kube-system
+  labels:
+    app: flux
+spec:
+  jobLabel: app.kubernetes.io/name
+  selector:
+    matchLabels:
+      app: flux
+  namespaceSelector:
+    matchNames:
+      - kube-system
```

Comment:
We might need to test with both a flux inside kube-system and an in a workload namespace